### ISSUE TITLE
Sublimelinter-eslint not triggered

### ISSUE THREAD

**Issue by mzagaja** [SublimeLinter/SublimeLinter-eslint#290]
Sublime Text 3.2.2
SublmeLinter 4.14.0
SublimeLinter-eslint 4.2.0

Tried:
```
// // // SublimeLinter Settings - User
{
  "debug": true,
  "linters": {
    "eslint": {
      "selector": "source.js"
    }
  }
}
```

Log output:
```
SublimeLinter: sublime_linter.py:324  Delay linting 'radar.js' for 0.0s
SublimeLinter: sublime_linter.py:340  No installed linter matches the view.
```

Show Scope Name:
`source.js meta.group.braces.curly.js`

```
mzagaja@MattZ-720 ~/D/digitalhub> which eslint
/usr/local/bin/eslint
mzagaja@MattZ-720 ~/D/digitalhub> eslint --version
v6.7.2
```

**Comment by kaste** [comment]
Ah, these fun moments when working with a computer.

- The settings look okay. Do you even need them, or is this part of trying to fix this problem?

- "No installed linter matches the view." is an early check, at this moment it doesn't matter if you have eslint actually installed. 

- Check if the plugin is enabled. `Package Control: Enable Package`.

- Open the Sublime console at the startup and look if something throws, and of course if this plugin actually loads. T.i.

```
reloading plugin SublimeLinter.__init__
...
reloading plugin SublimeLinter-eslint.linter
...
```

- In the console, e.g.: 

```
>>> import SublimeLinter
>>> SublimeLinter.lint.persist.linter_classes.keys()
```

will print all registered linters, as well. (But also the disabled linters.)

- Check, if you actually loaded a `.project` file with local overrides. 

```
>>> window.project_file_name()
>>> window.project_data()
```

- Or even a view override via

```
>>> view.settings().get('SublimeLinter.linters.eslint.disable')
```

🤞

**Comment by mzagaja** [comment]
It looks like `Package Control: Enable Package` was the winner on this. I did not expect it since I have no recollection of disabling it, but something must have. Thanks for the help!
